src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/TripleDES/TripleDESContractTests.cs

```diff
@@ -108,5 +108,18 @@ public static void Cfb8ModeCanDepadCfb64Padding()
                 Assert.Equal(new byte[] { 1, 2, 3, 4, 5 }, decrypted);
             }
         }
+
+        [Fact]
+        public static void SetKey_SetsKey()
```

Comment:
Consider extending the `TripleDesCannotSetWeakKey` test to include `SetKey` so we have coverage that TripleDES (and other algorithms with the concept of weak keys) rejects a weak key.